core/tests/test_batch_deskew_args.py

```diff
@@ -0,0 +1,57 @@
+# Tests for napari_lattice using arguments and saving output files as h5, as well as tiff
```

Comment:
Can we add a test to check the shape of the output image? and not only check if it exists?
We can save it as a 'tiff' file and read it back in using `skimage.io.imread`

Reply:
Do you want this in the CLI test, or when unit testing the functions? Also, what should the desired shape be?

Reply:
So, as an example

if we pass an array of shape (5,5,5) with parameters:
- angle of 30
- dx,dy and dz of value 1

the output shape should be (2,9,5) after deskewing..


Code to replicate it using just the main function from pyclesperanto library:

```
import numpy as np 
import pyclesperanto_prototype as cle

raw = np.zeros((5, 5, 5))

deskewed = cle.deskew_y(raw,angle_in_degrees=30,voxel_size_x=1,voxel_size_y=1,voxel_size_z=1)

deskewed.shape #(2,9,5)
```

Reply:
Perhaps as a unit test. 
don't worry about saving the file. 
Initialise lattice data class and check if  `self.deskew_vol_shape` matchess the desired output shape..

Reply:
https://github.com/BioimageAnalysisCoreWEHI/napari_lattice/blob/a17a04ea96d5e3593a65d6fe2fd884be3ad3babe/core/lls_core/lattice_data.py#L131